tensorflow_datasets/core/dataset_info.py

```diff
@@ -59,6 +63,12 @@
 from tensorflow_metadata.proto.v0 import schema_pb2
 from tensorflow_metadata.proto.v0 import statistics_pb2
 
+if typing.TYPE_CHECKING:
+  try:
+    import apache_beam as beam
+  except ImportError:
+    beam = Any
+
 # Name of the file to output the DatasetInfo protobuf object.
 DATASET_INFO_FILENAME = "dataset_info.json"
```

Comment:
Note you can add annotation on constant too.

Reply:
I have added `str` to every constant on top. Please check if it's correct.